index.js

```diff
@@ -19,9 +19,31 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    await Recipe.create({
+      title: "Hot Dog Fingers",
+      level: "UltraPro Chef",
+      ingredients: ["5 hot dogs", "5 human fingers", "100g of cut off nails"],
+      cuisine: "Anthropofagic",
+      dishType: "snack",
+      duration: 120,
+      creator: "Chef Hannibal",
+    });
+    console.log("Hot Dog Fingers");
   } catch (error) {
     console.log(error);
   }
+
+  await Recipe.insertMany(data);
+  data.forEach((Recipe) => console.log(Recipe.title));
+
+  await Recipe.findOneAndUpdate(
+    { title: "Rigatoni alla Genovese" },
+    { duration: 100 }
+  );
+  console.log("Great Success!");
+
+  await Recipe.deleteOne({ title: "Carrot Cake" });
+  console.log("Very Gut!");
 };
```

Comment:
The catch block needs to be at the end of the try block and not in the middle other wise it will only catch the error for the code above and not for the whole function:

```
(...)

  } catch (error) {
    console.log(error);
  }
};

manageRecipes();
```